tests/unit_tests/blas/batch/gemm_batch_stride.cpp

```diff
@@ -213,29 +282,49 @@ class GemmBatchStrideTests
         : public ::testing::TestWithParam<std::tuple<sycl::device *, oneapi::mkl::layout>> {};
 
 TEST_P(GemmBatchStrideTests, RealHalfPrecision) {
-    EXPECT_TRUEORSKIP(test<sycl::half>(std::get<0>(GetParam()), std::get<1>(GetParam()), 5));
+    EXPECT_TRUEORSKIP((test<sycl::half, sycl::half, sycl::half, sycl::half>(
+        std::get<0>(GetParam()), std::get<1>(GetParam()), 5)));
+}
+
+TEST_P(GemmBatchStrideTests, RealHalfRealScalarPrecision) {
```

Comment:
```suggestion
TEST_P(GemmBatchStrideTests, HalfHalfFloatPrecision) {
```
To be consistent with the test names in `gemm_usm.cpp`. More generally I think these test names should be more specific (see also `../extensions/gemm_bias.cpp`) and `Scalar` is not very helpful.